Cargo.lock

```diff
@@ -9281,7 +9281,7 @@ version = "1.5.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "3bfd5b7557925ce778ff9b9ef90e3ade34c524b5ff10e239c69a42d546d2af56"
 dependencies = [
- "rand 0.7.3",
+ "rand 0.3.23",
```

Comment:
this is unfortunate due to https://github.com/rust-lang/cargo/issues/5702, I can revert it manually